test/register/register.test.ts

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+import { ModeHandler } from "../../src/mode/modeHandler";
+import { setupWorkspace, cleanUpWorkspace, assertEqualLines } from '../testUtils';
+
+suite("register", () => {
+
+  let modeHandler: ModeHandler;
+
+  setup(async () => {
```

Comment:
Sure, feel free to comment on things I've missed ;-)

I will fix them tomorrow (3am here)


Get Outlook for Android






On Mon, Aug 1, 2016 at 2:57 AM +0430, "Grant Mathews" <notifications@github.com> wrote:





> @@ -0,0 +1,57 @@
> +"use strict";
> +
> +import { ModeHandler } from "../../src/mode/modeHandler";
> +import { setupWorkspace, cleanUpWorkspace, assertEqualLines } from '../testUtils';
> +
> +suite("register", () => {
> +
> +  let modeHandler: ModeHandler;
> +
> +  setup(async () => {

I would prefer these to be done in the style of newTest - I find that to be much easier to read. :)

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/VSCodeVim/Vim/pull/543/files/f3ecd4056c99d2f37571bb3bcc19c613e7e715ce#r72912330